modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -428,7 +428,12 @@ export default class LargeVideoManager {
             ReactDOM.render(
                 <Provider store = { APP.store }>
                     <I18nextProvider i18n = { i18next }>
-                        <PresenceLabel participantID = { id } />
+                        <PresenceLabel
+                            noContentStyles = { {
+                                className: 'presence-label no-presence'
+                            } }
```

Comment:
Passing in new objects as props can cause unnecessary reloads. Are you sure you want to pass in objects as props? Are you doing this so it's re-usable on web and native?

Reply:
Yes. Any ideas how to improve it?

Reply:
You can't make it work by passing in classnames directly then? Or keep it so the label can apply classnames but not break mobile?